gallery/scommanagedinstance-insights/Azure Monitor SCOM managed instance.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
mentioned several times in contributing docs, "types" other than workbooks are uncommon and controlled.  "scommmanagedinstance-insights" isn't a valid / known type, so if you are using it in code, this file would never be shown, and you'd be seeing errors in the javascript console that it isn't a known type and is falling back to workbooks.  if you need a new type, you need to work with the workbooks team to add it.  it isn't just a string, it is other labels, text, help links, support links, etc that go with it.